report_qweb_element_page_visibility/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+Report Qweb ElementPage Visibility
+==================================
+
+This module allows you to add 4 elements in QWEB reports:
+ - not-first-page: Shows element in every page but first
```

Comment:
Put a line break above to allow the correct rendering and remove the initial space.